independent-publisher-connectors/Proposal-Financial Industry Regulatory Authority FINRA (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,176 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "FINRA",
+    "description": "...",
```

Comment:
Kindly add the description.

Reply:
This was text from another connector, some placeholder content that's since been removed.